# <center> Парковка в городе Бирмингем</center>

Ссылка на данные: https://archive.ics.uci.edu/ml/machine-learning-databases/00482/

Данные, собранные на автостоянках в Бирмингеме, которыми управляет NCP, из городского совета Бирмингема. Показатели занятости (с 8:00 до 16:30) с 4 октября 2016 по 19 декабря 2016.

In [2]:
import pandas as pd 
parking =pd.read_csv("dataset.csv")

In [3]:
parking.head().T

0                    1  \
SystemCodeNumber          BHMBCCMKT01          BHMBCCMKT01   
Capacity                          577                  577   
Occupancy                          61                   64   
LastUpdated       2016-10-04 07:59:42  2016-10-04 08:25:42   

                                    2                    3  \
SystemCodeNumber          BHMBCCMKT01          BHMBCCMKT01   
Capacity                          577                  577   
Occupancy                          80                  107   
LastUpdated       2016-10-04 08:59:42  2016-10-04 09:32:46   

                                    4  
SystemCodeNumber          BHMBCCMKT01  
Capacity                          577  
Occupancy                         150  
LastUpdated       2016-10-04 09:59:48

In [4]:
parking.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 35717 entries, 0 to 35716
Data columns (total 4 columns):
SystemCodeNumber    35717 non-null object
Capacity            35717 non-null int64
Occupancy           35717 non-null int64
LastUpdated         35717 non-null object
dtypes: int64(2), object(2)
memory usage: 837.2+ KB


#### Количество столбцов: 4.
#### Количество строк: 35717.
#### Пропусков нет.
#### Информация о признаках:
SystemCodeNumber: идентификатор парковки, тип object.

Capacity: вместимость парковки, тип object.

Occupancy: заполняемость парковки, тип int64.

LastUpdated: дата и время измерения, тип int64.
#### Характеристики данных: Многомерные, одномерные, последовательные, временные ряды.
#### Применимы для задач классификации, регрессии, кластеризации.